### ISSUE TITLE
LRSchedulerPlugin: incompatibility with ReduceLROnPlateau

### ISSUE THREAD

**Issue by gab709** [ContinualAI/avalanche#692]
Hi! 
I tried to use torch.optim.ReduceLROnPlateau scheduler but an error occurs "_TypeError: step() missing 1 required positional argument: 'metrics'_". 
It happens because LRSchedulerPlugin calls the function step() of the scheduler without any argument. It's possible to adapt LRSchedulerPlugin to include ReduceLROnPlateau?
Thank you!

**Comment by AndreaCossu** [comment]
Hi @gab709 , you can subclass your strategy and specify custom operations on the optimizer. 
Take a look at the [`training_epoch`](https://github.com/ContinualAI/avalanche/blob/ca44d420bfd76d66a76ee33414214cbaed7ea31e/avalanche/training/strategies/base_strategy.py#L433) callback within `BaseStrategy`.

**Comment by gab709** [comment]
Thank you!
